jazz-ui/package-lock.json

```diff
@@ -1093,7 +1093,7 @@
       "integrity": "sha1-OciRjO/1eZ+D+UkqhI9iWt0Mdm8=",
       "dev": true,
       "requires": {
-        "hoek": "2.16.3"
+        "hoek": "4.2.1"
```

Comment:
The root parent that uses hoek should be upgraded in package.json and I don't see that change. Do you know the root parent in this case?

Reply:
Worked on it